src/coreclr/jit/importercalls.cpp

```diff
@@ -1485,6 +1542,35 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
         impPushOnStack(call, tiRetVal);
     }
 
+#ifdef SWIFT_SUPPORT
+    if (swiftErrorIndex >= 0)
+    {
+        // This Swift call uses the error register
+        assert(call->IsCall());
+        assert(call->AsCall()->unmgdCallConv == CorInfoCallConvExtension::Swift);
+        assert(swiftErrorIndex < (int)sig->numArgs);
+        CallArg* errorArg = call->AsCall()->gtArgs.GetArgByIndex(swiftErrorIndex);
+
+        assert(errorArg != nullptr);
+        GenTree* argNode = errorArg->GetNode();
+        assert(argNode != nullptr);
+
+        // SwiftError* arg should have been spilled to a local temp variable
+        assert(argNode->OperIs(GT_LCL_VAR));
+
+        // Store the error register value to where the SwiftError* points to
+        GenTree* errorRegNode = new (this, GT_SWIFT_ERROR) GenTree(GT_SWIFT_ERROR, TYP_REF);
+        errorRegNode->SetHasOrderingSideEffect();
```

Comment:
```suggestion
        errorRegNode->SetHasOrderingSideEffect();
        errorRegNode->gtFlags |= GTF_CALL | GTF_GLOB_REF;
```
Just to be be completely sure that this doesn't get reordered...